app/layout.js

```diff
@@ -0,0 +1,30 @@
+'use client'
+import React from "react";
+import "./globals.css";
+import { StateContext } from "@/context/StateContext";
+import Navigation from "@/layout/navigation";
+import { Inter } from "next/font/google";
+import PropTypes from "prop-types";
+import { Toaster } from "react-hot-toast";
+import RouteSecurity from "@/utilities/routeSecurity";
+const inter = Inter({ subsets: ["latin"] });
+
+export default function RootLayout({ children }) {
```

Comment:
- [ ]  Please, kindly refactor this function to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const RootLayout= ({ children }) => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪